[Service Now] Updating Service Catalog docs (#517)
hashicorp/web-unified-docs: content/terraform-docs-common/docs/cloud-docs/integrations/service-now/service-catalog-terraform/example-customizations.mdx

   the Flow then click the **Update**.
   ![Screenshot: ServiceNow Update Process Engine](/img/docs/servicenow-catalog-update-process-engine.png "Screenshot of updating Process Engine for the Catalog Item")

> ## Adjusting Polling Schedule Intervals

Reviewer: Suggestions for style to use sentence case and a gerund noun to start the header:
```suggestion
## Adjust polling schedule intervals
```

> Customers can customize the frequency at which Terraform-related updates are retrieved by adjusting polling schedule intervals.

Reviewer: We can directly address the user with "you" and suggestions for active voice:
```suggestion
You can customize how often ServiceNow retrieves Terraform-related updates by adjusting the polling schedule intervals.
```

There are three configurable polling intervals: 

> 1. **Worker Poll Apply Run**

Reviewer: I'm assuming these are all in the UI (so that's why they are bolded)?

1. **Worker Poll Destroy Workspace**
1. **Worker Poll Run State** 

> To modify these intervals, navigate to **All > Flow Designer > Flows** in ServiceNow. You may modify this frequency by changing

Reviewer: ```suggestion
To modify these intervals, navigate to **All > Flow Designer > Flows** in ServiceNow. Modify your polling frequency by changing
```

> the associated flow's Scheduled Trigger, allowing for more or less frequent updates in the ServiceNow ticket. 

Reviewer: Bolding because I assumer this is in the UI?
```suggestion
the associated flow's **Scheduled Trigger**, allowing for more or less frequent updates in the ServiceNow ticket. 
```

Author: yup
